src/invoices/InvoiceDetails/InvoiceActions/InvoiceActions.js

```diff
@@ -117,6 +119,18 @@ const InvoiceActions = ({
           </Icon>
         </Button>
       )}
+      {isCancelable && (
+        <IfPermission perm="ui-invoice.cancel">
```

Comment:
as it was described offline, it's not the best practice to use perm sets in IfPermission, please re-read how UI atom perm can be created and used within set (you can find it in our messages)

Reply:
take another look, please